src/photos.cs

```diff
@@ -1005,9 +1005,15 @@ interface PHPhotoLibrary {
 
 	[iOS (9,1)]
 	[TV (10,0)]
+#if MONOMAC
+	[DisableDefaultCtor] // NS_UNAVAILABLE
```

Comment:
Yep. :(

The iOS version is completely different - https://gist.github.com/chamons/377ae4c2cd6707bca9c18e46188c603e